WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -28,6 +28,18 @@ open class QuickStartTourGuide: NSObject {
         }
     }
 
+    func setup(for blog: Blog, withCompletedSteps steps: [QuickStartTour] = [], executeWithDelay: Bool = false) {
```

Comment:
Because this method is currently only ever used to add a delay (and without one it does the same as `setup(for:withCompletedSteps:)` I guess it could be renamed `delayedSetup(for:withCompletedSteps:)` or `setupWithDelay(for:withCompletedSteps:)` and then you could remove the guard entirely?

Reply:
Fixed! 2432d7d